database-mongodb/seed.js

```diff
@@ -1,26 +1,31 @@
-const database = require('./index.js');
-const Camp = require('./Camp.js');
 const faker = require('faker');
+const data = require('./index.js');
+const Camp = require('./Camp.js');
 
-var sampleData = [];
-for (var i = 0; i < 100; i++) {
-    var sampleObject = {
-        newId: i,
-        host: faker.name.findName(),
-        hostImage: faker.image.avatar(), 
-        body: faker.lorem.paragraphs(),
-        campers: [{ 
-            name: faker.name.firstName(), 
-            image: faker.image.avatar(), 
-            votes: faker.random.boolean()
-        }]
-     };
-     sampleData.push(sampleObject);
-};
+data.db.on('error', console.error.bind(console, 'connection error:'));
```

Comment:
I don't think you actually have to explicitly open your db here since calling .create at the bottom of the page will open the connection when it creates your new collection. Does this work without calling db.on and db.once?